Update README.md (#1)
betterplace/engineering_culture: README.md

## We have solid engineering standards

We are proud of the things we create, and use solid standards and best practices. At the moment those include:

> * Each pull request is reviewed by at least 2 developers before merging

Reviewer: ```suggestion
* Each pull request is reviewed by at least one other developer before merging
```
(i don't think 2 is realistic a.t.m. as there is often just one other dev who can judge the PR)

* Each functionality is covered by unit tests (and if we find missing tests while working on a story, we add them)
* The main steps of each user-facing feature are covered by integration tests
* We fix bugs first
* We automatically check for security issues and fix them as soon as feasible
* We automate recurring tasks to avoid mistakes

## We help each other grow

We are a team of people that help each other out and create great things together. Ways to help each other grow include:

* We take time for code reviews and to teach others what we know
* New approaches are discussed with the team so everybody can learn
* When we learn something new we share it (e.g. in our bi-weekly tech talks)

> * When stay respectful in our discussions

Reviewer: ```suggestion
* We stay respectful in our discussions
```

* We appreciate feedback that helps us improve, and point out and celebrate the achievements of others

## We feel responsible for our code and users

betterplace.org is built on trust, and we feel responsible for maintaining that trust. This means, for example:

* When something is broken, we feel responsible for fixing it (either by fixing it yourself, or finding someone who can do it)
* When working on code, we think of the next developer to work on it - it may be ourselves
* The goal of our work is to add value for our users
* We speak up when our values are in danger

## We are agile

We embrace the values of the [Agile Manifesto](https://agilemanifesto.org/) and believe in incremental change.

* We strive to deliver value in small increments
* We strive to have each increment be a shipable (as in: deployable) piece of working software

## We encourage experimentation

We know that in order to grow, we must allow space for experimentation and mistakes. To this end:

* We allow time for experimentation (e.g. through regular R&D)
* We learn from our mistakes, instead of assigning blame (e.g. through our blameless postmortems)

# Contributing

> All betterplace developers and engineers can create pull requests to update this document. Those PR can be edited by all of them. 

Reviewer: ```suggestion
All betterplace developers and engineers can create pull requests to update this document. Those PRs can be edited by all of them. 
```
